src/Phoenix.vue

```diff
@@ -39,17 +51,35 @@ export default {
   },
   computed: {
     ...mapState(['route', 'user']),
-    ...mapGetters(['configuration', 'activeNotifications', 'activeMessages', 'capabilities']),
+    ...mapGetters(['configuration', 'activeNotifications', 'activeMessages', 'capabilities', 'apps']),
     $_applicationsList () {
-      return this.configuration.applications
+      const list = []
+
+      // Get extensions manually added into config
+      list.push(this.configuration.applications)
+
+      // Get extensions which have at least one nav item
+      const navItems = this.$root.navItems
+      for (const item in navItems) {
+        list.push(this.apps[item])
+      }
+
+      return list.flat()
     },
 
     appNavigationEntries () {
       if (this.publicPage()) {
         return []
       }
+
+      const items = this.$root.navItems[this.currentExtension]
+
+      if (!items) {
+        return []
+      }
+
       // FIXME: use store or other ways, not $root
-      return this.$root.navItems.filter(item => {
+      return items.filter(item => {
```

Comment:
move the FIXME to above where this.$root is used ?

Reply:
Right. Done :wink: